clippy_lints/src/utils/hir_utils.rs

```diff
@@ -2,7 +2,7 @@ use crate::consts::{constant_context, constant_simple};
 use crate::utils::differing_macro_contexts;
 use rustc::hir::*;
 use rustc::lint::LateContext;
-use rustc::ty::TypeckTables;
+use rustc::ty::{self, Ty, TypeckTables};
```

Comment:
The import of `Ty` conflicts with the wildcard import from `hir::*`. If you need `ty::Ty`, you should change the wildcard import to an explicit import of the required `hir` modules.